### ISSUE TITLE
bootstrap 3 template

### ISSUE THREAD

**Issue by petrparolek** [iPublikuj/confirmation-dialog#4]
Dialog doesn't show with newest Twitter Bootstrap. Could someone help me please?

**Comment by akadlec** [comment]
Could you provide more info?

**Comment by premekkoch** [comment]
I have same trouble as @petrparolek. It's realy hard to find, what's going on there, but it seems like dialog automaticaly choose "No" answer and page is redirected back (or reload) before dialog shows on screen. Action response contains whole page original code.

**Comment by akadlec** [comment]
@premekkoch i think it is different. Could you please check your POST request if "yes" button is send to server?

**Comment by premekkoch** [comment]
I'm sorry, @akadlec - maybe I was wrong about my comment. Now I made clean instalation of nette 2.3.7, bootstrap 3.3.5 and confirmation-dialog 2.0.3 and it seem it working well (except of confirmation dialog isn't modal window). Maybe, when I tried it first, there was any "3rd party problem" or (more possible) I made some mistake on implementation. So, I take my first comment back, sorry about it and thanks for your patience.
